src/crypto/index.js

```diff
@@ -973,6 +965,25 @@ Crypto.prototype._getE2eRooms = function() {
     });
 };
 
+
+Crypto.prototype._onToDeviceEvent = function(event) {
+    try {
+        if (event.getType() == "m.room_key"
+            || event.getType() == "m.forwarded_room_key") {
+            this._onRoomKeyEvent(event);
+        } else if (event.getType() == "m.room_key_request") {
+            this._onRoomKeyRequestEvent(event);
+        } else if (event.isBeingDecrypted()) {
```

Comment:
Then I don't understand at what point we start decrypting these events. I guess they are being started before the event is emitted?